packages/generators/generators/lib/templates/ts/plugin-package.json.hbs

```diff
@@ -44,8 +44,8 @@
     "npm": ">=6.0.0"
   },
   "scripts": {
-    "develop": "tsc -w",
```

Comment:
I'd rather keep this as `develop`, because this is what we are using everywhere.